Lab1/Lab1/Services/Validation/ValidationFieldName.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Lab1.Services.Exceptions;
+
+namespace Lab1.Services.Validation
+{
+    public static class ValidationFieldName
```

Comment:
All your validations can be in single file. Just methods names can differ.

Reply:
Can i resolve this by creating interface and rework all Validation classes to realize this interface or only by making single Validation class with methods, that realize logics from all Validation classes?

Reply:
Interface don't accept static methods, rename all methods of Validation classes to name Validate.